# Generated by Django 4.2.3 on 2023-08-16 20:27

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('accounts', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='customuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'Ushbu nomli foydalanuvchi allaqachon mavjud. Boshqa nom tanlang'}, help_text='Majburiy. 150 yoki undan kam belgi. Harflar, raqamlar va ushbu belgilar @/./+/-/_.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
